int32 STARTED = 0
int32 STOPPED = 1
int32 DECODED = 2
int32 FAILED  = 3

Header header
int32 event
int32 utterance_id
transcript transcript
string audio_path
